.pre-commit-config.yaml

```diff
@@ -35,3 +35,15 @@ repos:
     rev: v2.3.2
     hooks:
       - id: prettier
+  - repo: local
+    hooks:
+      - id: djlint-templates
+        name: Lint project templates
+        language: python
+        files: tbxforms/.*\.html$
+        entry: djlint --profile django
```

Comment:
Is this needed? Profile is already defined in the conf: https://github.com/torchbox/tbxforms/pull/42/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R97